blocks/share-bar-block/features/share-bar/default.jsx

```diff
@@ -111,22 +95,19 @@ export const ShareBar = ({
 		if (customFields[social]) {
 			const encodedTitle = encodeURIComponent(headlineString);
 			const encodedUrl = encodeSocialUrl(websiteDomain, websiteUrl, encodedTitle, social);
-
 			const socialType = phrases.t(`global.social-${social.toLowerCase()}`);
 
 			shareButtons.push(
-				<button
+				<Button
+					className={`${BLOCK_CLASS_NAME}__${social.toLowerCase()}`}
```

Comment:
I'm not sure if this classname will work as expected. it looks like this would interfere with: 

```
const getLogoComponent = {
	facebook: <Icon name="Facebook" className={`${BLOCK_CLASS_NAME}__facebook`} />,
	linkedIn: <Icon name="LinkedIn" className={`${BLOCK_CLASS_NAME}__linkedin`} />,
	pinterest: <Icon name="Pinterest" className={`${BLOCK_CLASS_NAME}__pinterest`} />,
	twitter: <Icon name="Twitter" className={`${BLOCK_CLASS_NAME}__twitter`} />,
	email: <Icon name="Envelope" className={`${BLOCK_CLASS_NAME}__email`} />,
};

Reply:
```suggestion
```